src/test/java/org/utplsql/cli/TestRunCommandChecker.java

```diff
@@ -15,7 +15,7 @@ public void getCheckFailOnErrorMessage()
         assertNotNull(RunCommandChecker.getCheckFailOnErrorMessage(2, new Version("3.0.0")));
         assertNotNull(RunCommandChecker.getCheckFailOnErrorMessage(2, new Version("3.0.1")));
         assertNotNull(RunCommandChecker.getCheckFailOnErrorMessage(2, new Version("3.0.2")));
-        assertNull(RunCommandChecker.getCheckFailOnErrorMessage(2, new Version("3.0.3")));
+        assertNull(RunCommandChecker.getCheckFailOnErrorMessage(2, new Version("3.0.3.1266")));
```

Comment:
What's the reason for the change?

Reply:
Change in the API (where I put the exact numbers for the feature, which makes sense)